### DISCUSSION TITLE
Any ideas?

### DISCUSSION THREAD

**Discussion by Nodrex** [Nodrex/Testomania#166] [Ideas]
I have 2 :D

- App for GithubGist (I was able to find only 1 and it was ugly: https://play.google.com/store/apps/details?id=com.githubgist)
- I want to feed devices like Arduino/Raspberry from the wall socket, and also want to feed the power bank from this socket, and when electricity will be down use this power bank to feed devices, and when electricity will be up again feed devices and charge power bank also. basically UPS but for power banks and existing power banks. to create a `module for connecting existed power bank and Arduino/Raspberry`. this module also should be smart to stop charging the power bank when it will be full and start charging again when it fell below 90% not to overheat and overcharge the power bank. So this will be a hardware module, `a real electric device`. which we can ask some cheap china's manufacturer to produce and we can sell

**Comment by Nodrex** [comment]
@shalva97 
@kdiakonidze 
@likaGlonti 
@nmgalo 
What do you think guys?

**Reply by shalva97** [reply]
regarding the second part. I am already selling such item https://www.mymarket.ge/ka/pr/25771855/teqnika/audio-video-fototeqnika/datenvadi-elementebi-da-damtenebi/18650-liTiumis-elementi-da-type-c-damteni?OfferType=Basic&mode=preview :D

**Reply by Nodrex** [reply]
Woooow, so we can make it as a product, dude it is cool. Let's talk about it on Sunday. I believe I will be available this Sunday.
You know what we can try and use 3D Printer to print the shell for this product. what is missing here is `micro USB/type C` Output instead of wires.
Does it can check the level of charge and start charging if it falls below 90%?

**Reply by Nodrex** [reply]
I imagine that the final product has 1 input (mUSB/TC) from the wall socket and has 2 outputs (mUSB/TC) 1 for the device and the second for the power bank. The main key feature would be to monitor the charge level and start charging the power bank only on 90% and stop charging on 100%. Can we do that?

* mUSB - micro usb
* TC       - type c
